packages/e2e-tests/register/models/VerifySeedPhrase.js

```diff
@@ -0,0 +1,23 @@
+class VerifySeedPhrasePage {
+    constructor(page) {
+        this.page = page;
+    }
+    async navigate(accountId) {
+        await this.page.goto(`/setup-seed-phrase/${accountId}/phrase/verify`);
+    }
+    async getRequestedVerificationWordNumber() {
+        const wordNumberTextContent = await this.page.textContent(
+            "data-test-id=seedPhraseVerificationWordNumber"
+        );
+        return parseInt(wordNumberTextContent.slice(6));
+    }
+    async verifyWithWord(word) {
+        await this.page.fill(
+            "data-test-id=seedPhraseVerificationWordInput",
+            word
+        );
+        await this.page.click('[type="submit"]');
```

Comment:
As with `submitRecoveryOption()` case, would prefer if this was clicking on an element with an explicit ID